### DISCUSSION TITLE
Help to custom datalabel  across the chart

### DISCUSSION THREAD

**Discussion by HenriqueAurelio** [chartjs/chartjs-plugin-datalabels#286] [Q&A] [answered]
https://jsfiddle.net/o4kyt69j/2/

I want to do something like this 
![image](https://user-images.githubusercontent.com/31359644/136038136-d3a06008-b1e6-4ac7-821c-1d19c79ba0ee.png)
Write all values across the gauge instead of each arc but inside the chart js,

**Comment by LeeLenaleee** [comment, marked as answer]
You can set `align` en `anchor` to `'end'` to get them outside, also need some extra padding on the chart for this. This plugin cant make those tick marks afaik, you will need to draw those yourself on the canvas.

https://jsfiddle.net/Leelenaleee/f0wnhcjy/4/
